### ISSUE TITLE
Add SLO/alerts for VSCode extension installations

### ISSUE THREAD

**Issue by corneliusludmann** [gitpod-io/gitpod#6386]
@corneliusludmann Should we close it and open a new issue with SLO for extension installation and alert based on it?

_Originally posted by @akosyakov in https://github.com/gitpod-io/gitpod/issues/6190#issuecomment-950704203_

**Comment by akosyakov** [comment]
I cannot proceed since testing VS Code against Gitpod in prev envs is broken right now.

**Comment by loujaybee** [comment]
@akosyakov is there a related issue we can link this one to for unblocking it?

**Comment by akosyakov** [comment]
@loujaybee Actually it is unblocked now. @csweichel showed me how to switch to old image builder that i can change new changes. I will work on it again this week. Thank you for pointing!
